### PR TITLE
Add automation test for configuration

### PR DESCRIPTION
**Pull request by kongtiaowang**

### CODE REVIEW

File: modules/configuration/test/ConfigurationTest.php [hunk 1]
```diff
@@ -41,5 +42,97 @@ public function testConfigurationPageLoads()
...
+
+        $this->linkTest(
+            WebDriverBy::linkText("Study"),
+            $contentArea,
+            "Settings related to details of the study"
+        );
+        $this->linkTest(
+            WebDriverBy::linkText("Paths"),
+            $contentArea,
+            "pecify directories where LORIS-related files are stored or created."
```
Comment (driusan) [inline modules/configuration/test/ConfigurationTest.php]: There seems to be a typo here. I assume that should be "Specify"

File: test/integrationtests/LorisIntegrationTestForConfiguration.class.inc [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+/**
+ * Implementation of LorisIntegrationTest for configuration class.
+ *
+ * @category Test
+ * @package  Test
+ * @author   Wang Shen <wangshen.mcin@gmail.com> 
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+abstract class LorisIntegrationTestForConfiguration extends LorisIntegrationTest
```
Comment (driusan) [inline test/integrationtests/LorisIntegrationTestForConfiguration.class.inc]: Is this used/going to be used anywhere other than the ConfigurationTest? If not, maybe the function should just directly go in the ConfigurationTest class?

Comment (kongtiaowang) [inline test/integrationtests/LorisIntegrationTestForConfiguration.class.inc]: @driusan 
I created a test link method which I need to use many times. this class, I will add some new function or test data inside this class.

File: modules/help_editor/test/help_editorTest.php [hunk 3]
```diff
@@ -1,42 +0,0 @@
...
-     *
-     * @return void
-     */
-    function testHelpEditorEditHelpContentDoesPageLoad()
-    {
-        $this->safeGet($this->url . "/help_editor/edit_help_content/");
-        $bodyText = $this->webDriver->findElement(WebDriverBy::cssSelector("body"))->getText();
-        $this->assertContains("Edit Help Content", $bodyText);
-    }
-}
```
Comment (driusan) [inline modules/help_editor/test/help_editorTest.php]: Why does this file get deleted in this pull request?

**Pull request closed without merging**